core/config/initializers/13_rack_attack.rb

```diff
@@ -182,6 +182,25 @@ class Rack::Attack
     end
   end
 
+  # Throttle POST requests to /content_security_violations by IP address
+  #
+  # Key: "rack::attack:#{Time.now.to_i/:period}:csp_violations/ip:#{req.ip}"
+  throttle('csp_violations/ip', limit: 10, period: 10.minutes) do |request|
```

Comment:
Seems good, I'm curious about the thought to throttle this harder than the general throttle.

Reply:
In my head its likely that if violations were present on the site, a single user would hit the same violations over and over. So basically I set the limit to a number thats reasonable but stops the same violation reports from getting processed repeatedly.